### ISSUE TITLE
Log is not working

### ISSUE THREAD

**Issue by dirceu-jr** [kigster/puma-daemon#8]
Daemonizing using both ways leads to log not to work.  The file configured through `--redirect-stdout` stays with the content below:

```shell
=== puma startup: 2021-05-13 20:09:27 -0300 ===
[44739] *  Puma Daemon: Daemonizing (puma-daemon v0.1.2)...
```

Nothing more, and the file configured through `--redirect-stderr` with:

```shell
=== puma startup: 2021-05-13 22:19:47 +0000 ===
```

**Comment by kigster** [comment]
Can you please provide your puma config and the command line?

**Comment by dirceup** [comment]
> Can you please provide your puma config and the command line?

Yes,

Config: https://gist.github.com/dirceup/40cd59c9c6d579b03998ac54037b2ec6
Command line:
```
bundle exec dotenv puma -v -b \"unix:///var/project/shared/tmp/sockets/puma.sock\" -S /var/project/shared/tmp/sockets/puma.state --pidfile /var/project/shared/tmp/pids/puma.pid --control-url 'unix:///var/project/shared/tmp/sockets/pumactl.sock' --redirect-stdout \"/var/project/shared/log/puma.log\" --redirect-stderr \"/var/project/shared/log/puma-stderr.log\"
```

**Comment by ujinius** [comment]
Hello, I have same problem with logs. Is there any progress on it? Thanks.

**Comment by kigster** [comment]
Will take a look if I get a sense of the need for this gem....

**Comment by kigster** [comment]
Hopefully fixed with #11
